lib/request-handler.spec.js

```diff
@@ -33,16 +33,19 @@ describe('The request handler', function() {
   before(analytics.load);
 
   let camp;
+  let initialLongCache;
   beforeEach(function (done) {
     camp = Camp.start({ port: config.port, hostname: '::' });
     camp.on('listening', () => done());
+    initialLongCache = process.env.LONG_CACHE;
```

Comment:
Does this need to be reset before each test rather than just setting initially?

Reply:
I think it is better to set it back to the global/original value after each test. If you move this from `beforeEach()`/`afterEach()` to `beforeAll()`/`afterAll()`, some of the tests are going to be running with `LONG_CACHE` set to whatever value we set it to in the most recent test instead of the default value for the test environment (I realise at the moment this is probably `undefined`, but in principle this might change). This could introduce an unexpected behaviour. It is better if all tests in the suite inherit the default value for `LONG_CACHE` consistently. If a particular test needs it to be set to a particular value, we do it explicitly in that test.

Reply:
Yeah it makes sense to reset `process.env.LONG_CACHE = initialLongCache` `after/beforeEach()`,
But couldn't we set `const initialLongCache = process.env.LONG_CACHE`,
which would reset `LONG_CACHE` to its initial value after/before each test?

Reply:
I see what you mean now. Updated in a9d06bd